resources/views/volumes/videos/index/meta.blade.php

```diff
@@ -22,28 +22,38 @@
             </tr>
         @endif
         @if ($video->taken_at)
-            <tr>
-                <th>Created</th>
-                @if (is_array($video->taken_at))
-                    @foreach ($video->taken_at as $value)
-                        <td>{{ $value }}</td>
-                    @endforeach
-                @else
+            @if (is_array($video->taken_at))
+                <metadata-modal v-bind:show-modal="showModal" v-bind:times="times" v-bind:items="items" v-bind:name="name" v-on:load-modal="getTimes({{ collect($video->taken_at) }})" v-on:close-modal="hideMetadataModal"></metadata-modal>
+                <tr>
+                    <th>Created</th>
+                    <td>
+                        <button class="btn btn-default" type="button" title="Show full timestamps" v-on:click.prevent="showTimes()">Show values</button>
+                    </td>
+                </tr>
+                @foreach ($metadata as $field => $value)
+                    <tr>
+                        <th>{{ $metadataMap[$field] }}</th>
+                        @if (is_array($value))
+                            <td>
+                                <button class="btn btn-default" type="button" title="Show full metadata array" v-on:click.prevent="getMetadata({{ json_encode($metadataMap[$field]) }}, {{ collect($value) }})">Show values</button>
```

Comment:
I find the buttons a bit bulky. Maybe make the field name (e.g. "Created") clickable as a link element? Or just make "show values" a link instead of a button?

Reply:
No problems, I agree with all your comments :)

For the buttons I tried using field name as link elements, but it gives weird results with table's borders (at least on chrome, see below). So maybe the second option ? The thing is : if I use `class="btn btn-link"` the padding is not the same as for the rest of the table. So I tried with `class="btn-link"` and it works but is it a thing ?

```suggestion
                                <a class="btn-link" v-on:click.prevent="showMetadata({{ json_encode($field) }})">Show values</a>
```

![Capture](https://github.com/user-attachments/assets/04a62ad0-d6a1-4259-a8bb-f26df4f68bd9)

Reply:
Your screenshot looks like you removed the `<td>` elements completely. It should work if you change the text in the `<th>` to be a link and keep an empty `<td>` as next element.